### DISCUSSION TITLE
Kiosk development slowing down/pausing

### DISCUSSION THREAD

**Discussion by damongolding** [damongolding/immich-kiosk#31] [Announcements]
Hello,

So this morning I welcomed my second child into the world. So this a just a quick update to say that development on Kiosk will most likely pause until I'm not so sleep deprived and swimming in nappies/diapers...

See you all in the other side.

**Comment by SelfSufficient08** [comment]
Congrats sir.  Awesome project that I literally just stumbled onto and it works flawlessly in seconds.  I have been looking for something like this for so long.  I'll make a formal request then but does it have the ability to do search slideshows?  Like in immich you can search  "kids playing on the beach" and it will show you all photos that it thinks are that.  Is there a way to do a slideshow with that type of context.  If this is something you need to code and you have interest in developing to make happen, send me a link to donate to your efforts.  Again, great work !

**Comment by pelinoleg** [comment]
Сongratulations!

**Comment by damongolding** [comment]
Hey @SelfSufficient08!

Not in Kiosk's current form but I think a way to do it at the moment would be to search “kids playing on the beach” in Immich and then select all photos and add them to a new album. Grab the album ID and use that in Kiosk.

**Reply by SelfSufficient08** [reply]
Wasn't even expecting a response !  Thanks again !  Maybe when you are back at it, you could determine a way to do it more dynamically.  I like your idea but it would create a static library.  My dream is that for a birthday party or for a holiday, I could just call out "family thanksgiving" or "birthday" or "July" and so one.  then you get dynamic pictures that auto update.  
Seriously awesome product sir ! Enjoy your time with your little one !  There is light at the end of this phase. :-)

**Reply by damongolding** [reply]
Oh it's totally possible to implement that as an option/feature. 

I am able to reply to messages while holding my baby, coding on the other hand...not so much 😆

**Reply by SelfSufficient08** [reply]
Awesome !  I look forward to the potential !!

**Reply by soana** [reply]
Would be cool to have albums show up based on current date. Maybe have a list of significant dates that point to an album or a keyword search

**Comment by rushowr** [comment]
Welcome to the fatherhood deux club! Also, welcome to your temporary membership in the "infant caregiver/living dead" club. On average these memberships last about 6 months or so, and it'll be rough. Of course you know this, but then again if you're like me and had a 9 year gap, a reminder doesn't hurt.

Of course, you also know it's all worth it!

Great job with Immich Kiosk, and looking forward to see what comes after your temporary club membership expires! 😉🤙

**Comment by damongolding** [comment]
Thank you for the chuckle at 2:19am 😴

**Comment by lennon101** [comment]
Congratulations mate! My wife and I just welcomed our first kids into the world as well (twins!). 8 weeks old now and I can definitely relate to the lack of sleep 😂 great work on this project! I've got a few concepts/ideas that I'll put in the suggestion box. Thanks again for the great work!

**Reply by damongolding** [reply]
Massive congratulations and also my deepest sympathies!

Going from 0 kids to 1 was much harder for me then going 1 to 2. I can't (and I don't want to) imagine going from 0 to 2!

**Reply by lennon101** [reply]
The one thing that I have going for me is that I don't know any different! These are my first so I have no idea what it's like with just 1 😂

**Comment by joeldroid** [comment]
Congratulations Mate!

**Comment by jeprecated** [comment]
The second is harder on Dad! Thanks for the great companion app, and best of luck, mate

**Comment by soana** [comment]
any recommendation on how to control the display brightness based on the time of the day.
while current brightness of my monitor is OK during the daytime it looks a bit too bright during the night.

Thanks Damon for this great software and congrats!

**Reply by damongolding** [reply]
> any recommendation on how to control the display brightness based on the time of the day.
> 
> while current brightness of my monitor is OK during the daytime it looks a bit too bright during the night.
> 
> 
> 
> Thanks Damon for this great software and congrats! 

What display are you using? I am going to implement a sleep mode at  some point that just displays a black screen during the given time slot.

**Reply by soana** [reply]
It's a simple 14" HDMI driven by a raspberry pi in kiosk mode.

**Reply by damongolding** [reply]
> It's a simple 14" HDMI driven by a raspberry pi in kiosk mode.

Is the kiosk mode running a desktop environment or headless? I ask as you could use a cron job to turn the display off and on at certain times.

**Reply by soana** [reply]
I guess a cron job would work for me - let me investigate and if I find a nice way I will share here.

**Reply by damongolding** [reply]
> I guess a cron job would work for me - let me investigate and if I find a nice way I will share here.

Something like `xset dpms force off` could work maybe? You also might want to check this discussion out [https://github.com/damongolding/immich-kiosk/discussions/74](https://github.com/damongolding/immich-kiosk/discussions/74)

**Reply by soana** [reply]
for some reason no command I tried can open the display.
but the sleep mode works for me, thanks!

**Reply by damongolding** [reply]
> for some reason no command I tried can open the display.
> 
> but the sleep mode works for me, thanks!

So you're able to turn it off but not on?

**Reply by soana** [reply]
sorry for the confusing answer.
Trying to turn on/off the display using X11 or Wayland commands on the raspberry Pi did not work for me but, the new sleep feature added with: damongolding/immich-kiosk-development:0.11.0 works fine.

**Reply by damongolding** [reply]
> sorry for the confusing answer.
> 
> Trying to turn on/off the display using X11 or Wayland commands on the raspberry Pi did not work for me but, the new sleep feature added with: damongolding/immich-kiosk-development:0.11.0 works fine.


How odd. I'm going to set up my raspberry pi up on Monday using DietPi and I'll look into commands as it's something I will want to do. 

On a side note @soana , thank you for the support via the coffee button 🥰
